test/github.ts

```diff
@@ -1052,4 +1052,79 @@ describe('GitHub', () => {
       expect(changes!.get('missing-file')).to.not.be.undefined;
     });
   });
+
+  describe('createFileOnNewBranch', () => {
+    it('forks a new branch if the branch does not exist', async () => {
+      req = req
+        .get('/repos/fake/fake/git/ref/heads%2Fbase-branch')
+        .reply(200, {
+          object: {
+            sha: 'abc123',
+          },
+        })
+        .get('/repos/fake/fake/git/ref/heads%2Fnew-branch')
+        .reply(404)
+        .post('/repos/fake/fake/git/refs', body => {
+          snapshot(body);
```

Comment:
Does it make sense to assert with an inline object instead of using the snapshot? The snapshot is hard to follow.